test/FastTests/Issues/RavenDB-6250.cs

```diff
@@ -32,7 +32,8 @@ public void All_operations_has_details_providers()
                 Operations.OperationType.DatabaseExport,
                 Operations.OperationType.DatabaseImport,
                 Operations.OperationType.BulkInsert,
-                Operations.OperationType.IndexCompact
+                Operations.OperationType.IndexCompact,
+                Operations.OperationType.DatabaseRestore
```

Comment:
It is failing because we need to add DatabaseRestore to one of the HashSets
This check suppose to check if we Handled this operationType in the notification center